editor/components/table-of-contents/style.scss

```diff
@@ -5,6 +5,12 @@
 .table-of-contents__popover {
 	.components-popover__content {
 		padding: 16px;
+
+		@include break-small {
+			max-height: calc(100vh - 120px);
+			overflow-y: auto;
+			box-shadow: inset 0 -5px 5px -4px rgba( $dark-gray-900, .1 );
```

Comment:
Just noticed that adding the inset shadow to the Table of Contents has caused the outer shadow on the Popover to not appear.

This looks a little strange—the panel looks as though it's embedded into the page rather than floating on top of it:

<img width="688" alt="screen shot 2018-07-24 at 14 08 59" src="https://user-images.githubusercontent.com/612155/43116449-2817ec0a-8f4b-11e8-985a-0710267b1a81.png">